src/datasets/demo_replay.py

```diff
@@ -55,6 +55,7 @@ def create_demo_replay_data(env: BaseEnv,
         # Sample a trajectory
         traj_idx = rng.choice(len(demo_dataset.trajectories), p=weights)
         traj = demo_dataset.trajectories[traj_idx]
+        goal = train_tasks[traj.train_task_idx].goal
```

Comment:
lol i would have definitely messed up here and used `train_tasks[traj_idx]` which would have worked for now but possibly caused horrible bugs later in life